app/network_policies/bandwidthpolicydetailsctrl.ts

```diff
@@ -36,6 +36,8 @@ export class BandwidthPolicyDetailsComponent {
         /* Get particular Profile for based on key*/
         bandwidthPolicyDetailsCtrl.netprofilesModel.getModelByKey(activatedRoute.snapshot.params['key'],false,undefined)
             .then(function (policy) {
+                if (policy.DSCP === undefined) policy.DSCP = 0;
```

Comment:
If you set them as 0, netmaster doesn't save these fields in netprofile object and are not present when you do a GET.